### DISCUSSION TITLE
Use plot as blog post image

### DISCUSSION THREAD

**Discussion by 11rchitwood** [quarto-dev/quarto-cli#2319] [Q&A] [answered]
Let's say I have a post whose first code chunk is:

```r
plot(cars)
```

How can I make this be the preview image for the post?

**Comment by cscheid** [comment, marked as answer]
I don't believe this is supported right now, but that's a good idea. I'll create a github issue.

**Reply by cscheid** [reply]
Here's the [github issue](https://github.com/quarto-dev/quarto-cli/issues/2324). I'll lock this convo to keep the discussion in one place. Thanks for the idea!
